end_to_end/test_mistral.sh

```diff
@@ -18,4 +18,7 @@ gsutil -m cp -r gs://maxtext-external/mistral-7B-v0.1 /tmp
 python3 MaxText/llama_or_mistral_ckpt.py --base-model-path /tmp/mistral-7B-v0.1 --model-size mistral-7b --maxtext-model-path gs://maxtext-mistral/test/${idx}/decode-ckpt-maxtext/
 python3 MaxText/decode.py MaxText/configs/base.yml load_parameters_path=gs://maxtext-mistral/test/${idx}/decode-ckpt-maxtext/0/default run_name=runner_direct_${idx} per_device_batch_size=1 model_name='mistral-7b' tokenizer_path=gs://maxtext-external/mistral-7B-v0.1/tokenizer.mistral ici_tensor_parallelism=4 max_prefill_predict_length=4 max_target_length=16 prompt="I love to" autoregressive_decode_assert="read. I love to read about the Bible. I love" attention=dot_product
 
-# TODO(ranran): add training and fine-tuning tests
+# Training
+python3 MaxText/train.py MaxText/configs/base.yml load_parameters_path=gs://maxtext-mistral/test/${idx}/decode-ckpt-maxtext/0/default run_name=runner_${idx} per_device_batch_size=1 model_name='mistral-7b' ici_tensor_parallelism=4 steps=10
+
+# TODO(ranran): add fine-tuning tests
```

Comment:
there's no difference between fine-tuning and training fromo checkpoint in maxtext right now so I think you can delete the comment.

Reply:
Sg! I will remove them. Thanks!